sections/futures/Trade/MarketsDropdown.tsx

```diff
@@ -94,47 +97,32 @@ const MarketsDropdown: React.FC<MarketsDropdownProps> = ({ mobile }) => {
 	const selectedBasePriceRate = getBasePriceRate(marketAsset);
 	const selectedPastPrice = getPastPrice(marketAsset);
 
-	const getMinDecimals = React.useCallback(
-		(asset: string) => (isDecimalFour(asset) ? DEFAULT_CRYPTO_DECIMALS : undefined),
-		[]
-	);
-
 	const options = React.useMemo(() => {
-		return (
-			futuresMarkets?.map((market) => {
-				const pastPrice = getPastPrice(market.asset);
-				const basePriceRate = getBasePriceRate(market.asset);
-
-				return assetToCurrencyOption({
-					asset: market.asset,
-					key: market.marketKey,
-					description: getSynthDescription(market.asset, synthsMap, t),
-					price: formatCurrency(selectedPriceCurrency.name, basePriceRate, {
-						sign: '$',
-						minDecimals: getMinDecimals(market.asset),
-						suggestDecimals: true,
-					}),
-					change: formatPercent(
-						basePriceRate && pastPrice?.rate
-							? wei(basePriceRate).sub(pastPrice?.rate).div(basePriceRate)
-							: zeroBN
-					),
-					negativeChange:
-						basePriceRate && pastPrice?.rate ? wei(basePriceRate).lt(pastPrice?.rate) : false,
-					isMarketClosed: market.isSuspended,
-					closureReason: market.marketClosureReason,
-				});
-			}) ?? []
-		);
-	}, [
-		futuresMarkets,
-		selectedPriceCurrency.name,
-		synthsMap,
-		t,
-		getBasePriceRate,
-		getPastPrice,
-		getMinDecimals,
-	]);
+		return futuresMarkets.map((market) => {
+			const pastPrice = getPastPrice(market.asset);
+			const basePriceRate = getBasePriceRate(market.asset);
+
+			return assetToCurrencyOption({
+				asset: market.asset,
+				key: market.marketKey,
+				description: getSynthDescription(market.asset, synthsMap, t),
+				price: formatCurrency(selectedPriceCurrency.name, basePriceRate, {
```

Comment:
formatDollars

Reply:
Nice catch. Switched this to use formatDollars now.